dockerbuild/wait_for_mongodb.sh

```diff
@@ -8,13 +8,12 @@ set -e
 thrift_version="$1"
 host="$2"
 
-# 'foo' doesn't matter here. Just needs something so that $host is interpreted
-# as a hostname and not a database name.
-until mongo "$host"/foo --eval "db.getCollectionNames()" > /dev/null 2>&1; do
+export PASSWORD=$(< /mongo/mongodb-password)
+
+until pgrep mongod > /dev/null 2>&1; do
```

Comment:
This doesn't look correct. Won't this fail in docker-compose?

Reply:
I looked up a way to test if there is connection to the mongo server and the web said this should work. it didn't fail with docker-compose.

Reply:
My concern is that it relies on their being a process named `mongod` in the current container. This shouldn't normally happen...

Reply:
okay yeah that's fair! Changed it back to the original check.